include/oneapi/dpl/internal/distributed_ranges_impl/concepts/concepts.hpp

```diff
@@ -0,0 +1,70 @@
+// SPDX-FileCopyrightText: Intel Corporation
+//
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I don't know a lot about SPDX, it seems it is a way to generate documentation, headers, license information, in a way which allows you to not have lots of redundant copies of the same text.  This makes sense, but I have questions from a logistical and legal perspective. 

Since oneDPL is developed in the open, and one of the mechanisms of getting and using oneDPL is by building directly from this repository: 
Is this a valid way of specifying copyright text and license publicly, or does it require passing through some software which generates these things to be legal and meet the requirements?

Do we need any additional steps in our release process to accommodate this?

Reply:
> Since oneDPL is developed in the open, and one of the mechanisms of getting and using oneDPL is by building directly from this repository: Is this a valid way of specifying copyright text and license publicly, or does it require passing through some software which generates these things to be legal and meet the requirements?

Thank you for rising this issue. I've just changed in https://github.com/oneapi-src/oneDPL/pull/1687 all SPDX copyright comments in Distributed Ranges to be exactly the same as the ones you use in oneDPL. So we do not need to determine whether this was valid way or not.